docs/src/main/sphinx/sql/show-functions.md

```diff
@@ -18,11 +18,18 @@ For each function returned, the following information is displayed:
 - Deterministic
 - Description
 
+Use the `FROM` or `IN` keyword to list [user-defined
```

Comment:
This is actually not limited to UDFs. It can be any functions in a specific catalog.

Reply:
Reworded.